.github/workflows/instance-tests.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Snyk Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: instance-tests
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  instance-tests:
+    strategy:
+      matrix:
+        environment: [Standard, MT1, MT2]
+    name: instance tests
+    environment: ${{matrix.environment}}
```

Comment:
Maybe we can save computation resources and specify environment only for "Run Instance Tests" task? 😉